examples/test_scr_poststage

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+#
+# This script generates checkpoints on the IBM burst buffer disk, and then
+# starts transferring them to GPFS before exiting.  Then, the scr_poststage
+# script gets run after the transfers finish, and finalizes them.
+#
+# Example:
+#
+# # Allocate one node with 100GB burst buffer, run test_scr_poststag to
+# # create and begin transferring checkpoints, and after the transfers finish
+# # (after the job ends), call scr_poststage to finalize them.
+# bsub -nnodes 4 -stage storage=10:out=,scr_poststage test_scr_poststage
+#
+
+# Where do you want to write your checkpoints to?  This should be somewhere in
+# on the burst buffer filesystem.
+
+# Where do you want your checkpoints to transfer to in GPFS?
+GPFS_DIR="/p/gpfs1/$(whoami)"
+
+if [ -z "$BBPATH" ] ; then
+    echo "BBPATH isn't set in the script, please set it"
+    exit
+fi
+
+if [ -z "$GPFS_DIR" ] ; then
+    echo "Please set \$GPFS_DIR in test_scr_poststage"
+    exit
+fi
+
+# Absolute path to SCR's 'test_api' and 'test_multiple' binaries
+BINDIR=""
+
+if [[ ! -x $BINDIR/test_api ]] ; then
+    echo "Please add the absolute path to SCR's 'test_api' binary into BINDIR in this script"
+    exit
+fi
+
+function make_scr_conf {
+    echo "
+SCR_COPY_TYPE=FILE
+
+SCR_FLUSH=1
+SCR_FLUSH_ASYNC=1
+SCR_FLUSH_POSTSTAGE=1
+SCR_CACHE_BYPASS=0
+
+# Make our burst buffer a cache device
+STORE=$BBPATH GROUP=NODE COUNT=1 FLUSH=BBAPI
+
+# Just need this entry for SCR_CNTL_BASE
+STORE=/dev/shm GROUP=NODE
+
+CKPT=0 INTERVAL=1 GROUP=NODE   STORE=$BBPATH TYPE=PARTNER
+
+SCR_CNTL_BASE=/dev/shm
```

Comment:
Let's try to take this out one more time.  I merged a change that should now create a store descriptor for the control directory by default.  You will likely need to rebase on the latest develop to pick it up.